src/pages/index.js

```diff
@@ -1,5 +1,12 @@
 import "./index.css";
-import { enableValidation,settings } from "../scripts/validation";
+import {
+  enableValidation,
+  settings,
+  resetValidation,
+  disableButton,
+} from "../scripts/validation";
+import Api from "../utils/Api.js";
+import { setButtonText } from "../utils/helpers.js";
 
 const initialCards = [
```

Comment:
From the 8th sprint all constants (including `initialCards` and `settings`) should be in `constants.js` in the `utils` folder to clean up the main file.